python/lsst/ts/externalscripts/auxtel/latiss_wep_align.py

```diff
@@ -249,7 +249,7 @@ def get_donut_catalog(result: Struct, wcs: SkyWcs) -> pandas.DataFrame:
 
 def get_image(
     data_id: typing.Dict[str, typing.Union[int, str]],
-    best_effort_isr: BestEffortIsr,
+    best_effort_isr: typing.Any,
```

Comment:
.... I honestly don't know what this does...

Reply:
So, the problem is that `BestEffortIsr` is only available if `summit_utils` is available, and python fails to create the class when it is not. Adding a `typing.Any` is basically telling the type checker (which is external to python) that this object can be of any type.. I'll fix it back in the future once I can incorporate summit_utils into the tssw stack.